spec.md

```diff
@@ -1082,10 +1082,22 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
+| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the update request, even if the service is updated asynchronously. If present, MUST be a non-empty string. |
 | operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Last Operation](#polling-last-operation) endpoint in a percent-encoded query parameter. If present, MUST be a non-empty string. |
 
+The ability to include updated metadata in the response message was
+added in version 2.14. Any Platform sending an API Version Header with a
+value of 2.14 or greater MUST use the new values specified in the response
```

Comment:
@n3wscott @duglin This seems to conflict with the compatibility matrix? We are saying platforms MUST support this in v2.14 and later.

Reply:
Yeah it should be added, and that is why we need to commit the matrix PR :D